web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -36,7 +36,7 @@ function AccountSelect({ ...props }) {
       </PopupModalHeader>
       <PopupModalContent>
         <div className="overflow-scroll h-80">
-          {filteredAccounts.map((filteredAccount: AccountObject) => (
+          {accounts.map((filteredAccount: AccountObject) => (
```

Comment:
```suggestion
          {accounts.map((filteredAccount: AccountObject, index: number) => (
```

`.map` method always have a `key`. So, can you add an `index` value and put this `index` in `Selector` props ? 🤔 
Like this :

```typescript
<Selector
    customClass="pb-4"
    key={index}
    preIcon={<Identicon username={filteredAccount.nickname} />}
    posIcon={<MassaLogo size={24} />}
    content={filteredAccount.nickname}
    variant="secondary"
    amount={formatStandard(
      +filteredAccount.candidateBalance / 10 ** 9,
    )}
    onClick={() => handleSetRecipient(filteredAccount)}
    />
```